src/components/specimen/components/contentBlocks/specimenBlock/References.tsx

```diff
@@ -0,0 +1,106 @@
+/* Import Dependencies */
+import { Card, Row, Col } from 'react-bootstrap';
+
+/* Import Icons */
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faLink } from '@fortawesome/free-solid-svg-icons';
+
+
+const References = () => {
```

Comment:
This one looks a bit strange to me and contains too much information. For both references, linkages and assertions we could have dozens of those. I think this needs more refinement.

Reply:
Yes this needs more refinement, was puzzling too much what could be displayed in here (e.g. the links or statistics), did not want to waste too much time on it because I think Wouter can give a clearer interpretation of this

Reply:
Agree, let's do it in some next refinement session :+1: